covidprognosis/data/nih_chest_xrays.py

```diff
@@ -38,8 +38,10 @@ def __init__(
         transform: Optional[Callable] = None,
         resplit: bool = False,
         resplit_seed: int = 2019,
-        resplit_ratios: List[float] = [0.7, 0.2, 0.1],
+        
```

Comment:
resplit_ratios: List[float] = None,

sorry i forgot to add this line thank you